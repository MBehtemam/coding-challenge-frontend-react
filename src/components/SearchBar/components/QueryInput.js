import styled from 'styled-components';

const QueryInput = styled.input`
  background-clip: padding-box;
  background-color: #fff;
  border: 1px solid #ced4da;
  border-radius: 0.25rem;
  color: #495057;
  display: block;
  font-size: 1rem;
  line-height: 1.5;
  padding: 0.375rem 0rem;
  transition: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
  width: 90%;
`;
export default QueryInput;
