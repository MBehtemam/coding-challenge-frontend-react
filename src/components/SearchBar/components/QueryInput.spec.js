import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import QueryInput from './QueryInput';

describe('Test suits for Query Input', () => {
  it('it should match with snapshot', () => {
    const tree = renderer.create(<QueryInput />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
