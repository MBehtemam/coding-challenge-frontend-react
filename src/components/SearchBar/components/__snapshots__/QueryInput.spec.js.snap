// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for Query Input it should match with snapshot 1`] = `
.c0 {
  background-clip: padding-box;
  background-color: #fff;
  border: 1px solid #ced4da;
  border-radius: 0.25rem;
  color: #495057;
  display: block;
  font-size: 1rem;
  line-height: 1.5;
  padding: 0.375rem 0rem;
  -webkit-transition: border-color 0.15s ease-in-out,box-shadow 0.15s ease-in-out;
  transition: border-color 0.15s ease-in-out,box-shadow 0.15s ease-in-out;
  width: 90%;
}

<input
  className="c0"
/>
`;
